src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to make new metrics available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set of multiple [Prometheus](https://prometheus.io/) shards in `agent mode` functioning as monitoring agents to be able to collect and forward cluster critical and workload metrics to a central [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
```

Comment:
Probably nitpicking but shouldn't we call it `kube-prometheus-stack-operator` now instead of `prometheus-operator` ?

Reply:
No because the operator is still the name :D